### PR TITLE
Fix isRamAlmostOverloaded function in Consumer

### PR DESCRIPTION
**Pull request by agentsib**

### CODE REVIEW

**Comment by agentsib** [comment]
Hello, maybe need change something?

File: RabbitMq/Consumer.php [hunk 1]
```diff
@@ -175,6 +175,6 @@ protected function isRamAlmostOverloaded()
     {
         $memoryManager = new MemoryConsumptionChecker(new NativeMemoryUsageProvider());
 
-        return $memoryManager->isRamAlmostOverloaded($this->getMemoryLimit(), '5M');
+        return $memoryManager->isRamAlmostOverloaded($this->getMemoryLimit().'M', '5M');
```
Comment (stloyd) [inline RabbitMq/Consumer.php]: We need to check & adjust if already set memory limit is not in such format.

Comment (adm-bome) [inline RabbitMq/Consumer.php]: This check is already done by convertHumanUnitToNumerical() method.

**Just a suggestion:** 
Alter/better documentation in the ReadMe
Its unclear  the `memory-limit` / `-l` option is **not** in MB's by default.

The example `-l 256` is 256 bytes and not 256 mega bytes.
so for reference `-l=256M` will be the correct notation or `-l=268435456`

**Comment by stloyd** [comment]
Thanks @agentsib !

File: RabbitMq/Consumer.php (hunk 1 above)
Comment (agentsib) [inline RabbitMq/Consumer.php]: @adm-bome Ok, I'll do it as soon as I have time. Thanks/

**Pull request merged**